parts/kubernetesmastercustomscript.sh

```diff
@@ -165,6 +167,11 @@ function writeKubeConfig() {
     chmod 700 $KUBECONFIGDIR
     chmod 600 $KUBECONFIGFILE
 
+    FQDNSuffix="cloudapp.azure.com"
+    if [ "$TARGET_ENVIRONMENT" = "AzureChinaCloud" ]
```

Comment:
I think you're right. It looks like in Windows we only ever points at an IP address or localhost, never the external DNS name... Not sure why I was worried about this. I think you're right, nothing to worry about here.